recipes/libgfortran-abi/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.5.0" %}
+
+package:
+  name: libgfortran-abi
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+
+requirements:
+  run:
+    - libgfortran 3.0.0
```

Comment:
I think the best way ultimately will be some jinja magic that inspects the output of strings, or otherwise introspects the library.  For the time being, we're stuck with manual updates.  Tests based on strings output are probably easier to get right than getting things in on the front side of things.